microservices/product/src/commands/services.py

```diff
@@ -120,7 +120,7 @@ async def get_products(request: Request) -> Response:
 
     @staticmethod
     @enroute.broker.command("GetProduct")
-    @enroute.rest.command("/products/{uuid}", "GET")
+    # @enroute.rest.command("/products/{uuid}", "GET")
```

Comment:
Is this comment on purpose or is it a mistake?

Reply:
Hi @vladyslav-fenchak! In this case was intentional (at least for the moment), because the `/products/without-stock` endpoint collided with this one. In any case, I'll try to define a regex for the `uuid` parameter in order to avoid that conflict. 🙂